pupil_src/shared_modules/background_helper/base.py

```diff
@@ -0,0 +1,48 @@
+"""
+(*)~---------------------------------------------------------------------------
+Pupil - eye tracking platform
+Copyright (C) 2012-2019 Pupil Labs
+
+Distributed under the terms of the GNU
+Lesser General Public License (LGPL v3.0).
+See COPYING and COPYING.LESSER for license details.
+---------------------------------------------------------------------------~(*)
+"""
+
+import abc
+import typing
+
+
+class EarlyCancellationError(Exception):
+    pass
+
+
+class Task_Proxy(abc.ABC):
```

Comment:
1. Remove underscore from class names

2. Why is this not called `BaseTaskProxy`? Same with the 2 different implementations (threaded vs multiprocessing). I see that you are exposing them with different names through the module's `__init__.py`, why not name them differently in the first place? This makes it a lot easier to search for them as well.